rskj-core/src/main/java/co/rsk/remasc/RemascTransaction.java

```diff
@@ -28,7 +28,19 @@
 import static org.ethereum.rpc.TypeConverter.toUnformattedJsonHex;
 
 /**
+ * <p>
  * Tx that invokes Remasc's processMinersFees method.
+ * </p>
+ * <p>
+ * If you would like an easier to know if a transaction is a Remasc transaction rather than using
```

Comment:
It is not clear to me when should I use one or the other 😕

Reply:
No concerns. I just think this could be explained in the docs:

- Use `Transaction.isRemascTransaction` if you have not yet parsed the transaction and it is still in binary format. 
- Use `instance of` to know if a parsed transaction is a remasc transaction.

Note that `instance of` may return true even though `isRemascTransaction` is false... this is the most confusing case and the one I think could be clearer.

Reply:
Thanks for the explanation, updated the PR, please let me know if that works.